control-plane/pkg/reconciler/sink/kafka_sink_test.go

```diff
@@ -96,8 +95,9 @@ var (
 		Path:   fmt.Sprintf("/%s/%s", SinkNamespace, SinkName),
 	}
 
-	createTopicError = fmt.Errorf("failed to create topic")
-	deleteTopicError = fmt.Errorf("failed to delete topic")
+	errCreateTopic = fmt.Errorf("failed to create topic")
+
+	errorDeleteTopic = fmt.Errorf("failed to delete topic")
```

Comment:
These two variables should probably be named consistently
```suggestion
	errorCreateTopic = fmt.Errorf("failed to create topic")

	errorDeleteTopic = fmt.Errorf("failed to delete topic")
```

Reply:
Oh okay, @Leo6Leo should we rename `errorDeleteTopic` to be `errDeleteTopic` as well then?